ui/glean/pings.yaml

```diff
@@ -0,0 +1,14 @@
+---
+# Schema
+$schema: moz://mozilla.org/schemas/glean/pings/2-0-0
+
+classification:
+  description: >
+    A ping to record treeherder classification data
+  include_client_id: false
```

Comment:
Out of curiosity, why do you not want a client id?

Reply:
I only care about counting number of transactions and metrics from treeherder- I don't care who does it or where they are from or what type of computer they are on.